conjuring-rules.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="/CSS_Components/styles.css">
+
+    <link rel="stylesheet" href="/CSS_Components/rules.css">
+
+    <title>Rules</title>
+
+    
+
+</head>
+<body>
+    <!-- header container -->
+    <div class="header-content">
+        TRIVIA WITH TUSHAR
+    </div>
+
+    <!-- rules body -->
+     <div class="rules-container">
+
+        <ul>
+            <h3 class="rules-header">Rules for the Quiz:</h3>
+            
+            <li>
+                You have a total of 5 questions.
+            </li>
+
+            <li>
+                Only of of the options is correct.
+            </li>
+
+            <li>
+                10 marks is awarded for every correct answer.
+            </li>
+
+            <li>
+                There is no negative marking.
+            </li>
+        <div class="start-btn-container">
+            <a href="/conjuring.html">
+                <button class=" play-btn">
+                    Start
+                </button>
+            </a>
+        </div>
+            
+        </ul>
+        
+     </div>
+    
```

Comment:
Can beautify file and remove unwanted spacing

Reply:
Done.